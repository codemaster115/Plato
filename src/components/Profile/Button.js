import styled from 'styled-components';

export default styled.div`                 
    box-sizing: border-box;    
    height: 30px;
    width: 94px;        
    background: none rgb(85, 90, 191);
    border-radius: 4px;
    box-shadow: rgb(238, 241, 244) 0px 2px 7px 0px;
    cursor: pointer;
    color: white;
    float: right;    
    font-family: "Source Sans Pro";
    font-size: 14px;
    font-weight: 400;
    padding: 8px 11px;
    &:hover {
      opacity: 0.9;
    }
`;