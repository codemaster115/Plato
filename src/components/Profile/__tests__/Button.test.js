import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Button from '../Button';

test('Compare snapshots', () => {
  const Component = renderer.create(<Button />);

  const Tree = Component.toJSON();
  expect(Tree).toMatchSnapshot();
});
test('Renders without crashing', () => {
  mount(<Button />);
});