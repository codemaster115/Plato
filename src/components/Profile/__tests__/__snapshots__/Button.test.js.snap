// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compare snapshots 1`] = `
.c0 {
  box-sizing: border-box;
  height: 30px;
  width: 94px;
  background: none rgb(85,90,191);
  border-radius: 4px;
  box-shadow: rgb(238,241,244) 0px 2px 7px 0px;
  cursor: pointer;
  color: white;
  float: right;
  font-family: "Source Sans Pro";
  font-size: 14px;
  font-weight: 400;
  padding: 8px 11px;
}

.c0:hover {
  opacity: 0.9;
}

<div
  className="c0"
/>
`;
